spiketoolkit/postprocessing/postprocessing_tools.py

```diff
@@ -562,6 +614,102 @@ def set_unit_properties_by_max_channel_properties(recording, sorting, property,
                                                  ms_before=ms_before, ms_after=ms_after, verbose=verbose)
             sorting.set_unit_property(unit_id, property, recording.get_channel_property(max_chan, property))
 
+def get_firing_times_ids(sorting, sampling_frequency):
```

Comment:
I think this should go to the validation motule since it's a preparation function for the quality metrics

Reply:
Agree

Reply:
:+1: